airbyte-integrations/connectors/destination-jdbc/src/main/resources/spec.json

```diff
@@ -1,6 +1,8 @@
 {
   "documentationUrl": "https://docs.airbyte.io/integrations/destinations/postgres",
   "supportsIncremental": true,
+  "supportsNormalization": true,
+  "supportsDBT": true,
```

Comment:
I would set both of these to false for the generic jdbc destination

Then, it is overridden in each destination such as postgres etc anyway